### ISSUE TITLE
Pass -fvisibility=hidden when compiling C++ proto runtime

### ISSUE THREAD

**Issue by indragiek** [bazelbuild/rules_proto#41]
Is there a way to pass additional flags for compiling the C++ proto runtime? I'm trying to pass `-fvisibility=hidden` to avoid exporting symbols, but cannot find an obvious way to do so. Wrapping a `cc_proto_library` in a `cc_library` with the flag set doesn't seem to do anything, because the flag isn't passed transitively to the dependencies.

**Comment by Yannic** [comment]
I don't think this is possible. Would it be possible to add a custom linker script that hides the symbols from the proto runtime?

**Comment by indragiek** [comment]
@Yannic I'll give that a try, thanks.
